sdk/eventhub/azure-eventhubs/azure/eventhub/sender.py

```diff
@@ -300,6 +255,67 @@ def send(self, event_data):
         else:
             return self._outcome
 
+    def send(self, event_data):
+        """
+        Sends an event data and blocks until acknowledgement is
+        received or operation times out.
+
+        :param event_data: The event to be sent.
+        :type event_data: ~azure.eventhub.common.EventData
+        :raises: ~azure.eventhub.common.EventHubError if the message fails to
+         send.
+        :return: The outcome of the message send.
+        :rtype: ~uamqp.constants.MessageSendResult
+
+        Example:
+            .. literalinclude:: ../examples/test_examples_eventhub.py
+                :start-after: [START eventhub_client_sync_send]
+                :end-before: [END eventhub_client_sync_send]
+                :language: python
+                :dedent: 4
+                :caption: Sends an event data and blocks until acknowledgement is received or operation times out.
+
+        """
+        if self.error:
+            raise self.error
+        if event_data.partition_key and self.partition:
+            raise ValueError("EventData partition key cannot be used with a partition sender.")
+        event_data.message.on_send_complete = self._on_outcome
+        return self._send_event_data(event_data)
+
+    def send_batch(self, batch_event_data):
+        """
+        Sends an event data and blocks until acknowledgement is
+        received or operation times out.
+
+        :param event_data: The event to be sent.
```

Comment:
Doc string needs updating

Reply:
Yes, we have an issue to review all docstrings after code is relatively stable